.travis.yml

```diff
@@ -30,16 +30,26 @@ gemfile:
   - gemfiles/rails50.gemfile
 matrix:
   exclude:
-    - rvm: 1.9.2
-      gemfile: gemfiles/mongoid3.gemfile
-    - rvm: 1.9.2
-      gemfile: gemfiles/rails4.gemfile
     - rvm: 1.9.3
       gemfile: gemfiles/rails50.gemfile
     - rvm: 2.0.0
       gemfile: gemfiles/rails50.gemfile
     - rvm: 2.1.8
       gemfile: gemfiles/rails50.gemfile
+    - rvm: 2.4.0
+      gemfile: gemfiles/mongoid2.gemfile
+    - rvm: 2.4.0
+      gemfile: gemfiles/mongoid3.gemfile
+    - rvm: 2.4.0
+      gemfile: gemfiles/mongoid4.gemfile
+    - rvm: 2.4.0
+      gemfile: gemfiles/rails3.gemfile
+    - rvm: 2.4.0
+      gemfile: gemfiles/rails4.gemfile
+    - rvm: 2.4.0
+      gemfile: gemfiles/rails41.gemfile
+    - rvm: 2.4.0
+      gemfile: gemfiles/rails42.gemfile
```

Comment:
Only Rails 4.2 and 5.0 are still supported (http://guides.rubyonrails.org/maintenance_policy.html).

I've updated the PR to also run tests on Rails 4.2 and Ruby 2.4.

All modern Rubies ship JSON as part of stdlib, an error occurs when installing `json` gem provided by `activesupport` on older versions.

It's not possible to fix version of json to `2.0` in Gemfile because ActiveSupport is requiring `json (>= 1.7.7, ~> 1.7)`.